bin/benchpark

```diff
@@ -163,7 +215,21 @@ def benchpark_setup(subparsers, actions_dict):
         help="Where to install packages and store results for the experiments. Benchpark expects to manage this directory, and it should be empty/nonexistent the first time you run benchpark setup experiments.",
     )
 
+    system_create_parser = subparsers.add_parser(
+        "system-create", help="Initialize a system config"
```

Comment:
It'd be great if we could split this out into `system create` instead and support additional sub-commands such as `system list` and `system edit` in the future.